UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Helpers/ISerializer.cs

```diff
@@ -1,18 +1,18 @@
-using System;
-using System.Globalization;
-using PlayFab.Internal;
-
-namespace PlayFab.Json
+﻿namespace PlayFab.MultiplayerAgent.Helpers
 {
-    public class SimpleJsonInstance : ISerializerPlugin
+    using System;
+    using System.Globalization;
+    using System.Reflection;
+
+    public class SimpleJsonInstance
```

Comment:
>SimpleJsonInstance [](start = 17, length = 18)

Nit: The name of this class doesn't quit match the file name.

Reply:
nice catch, fixing!